.github/workflows/release.yml

```diff
@@ -55,7 +56,7 @@ jobs:
       ci-access-token: ${{ secrets.CI_ACCESS_TOKEN }}
 
   create-compute-release-branch:
-    if: inputs.create-compute-release-branch
+    if: ${{ github.event.schedule == '0 6 * * FRI' || inputs.create-compute-release-branch }}
```

Comment:
I'm afraid this will conflict with the Storage release at the same time. 
We need either to remove an extra line in the schedule or slightly change the time for a Compute release

Reply:
7 AM works as well :) Thanks!